relay-server/src/services/spooler/spool_utils.rs

```diff
@@ -0,0 +1,33 @@
+//! Contains helper utils which help to manage the spooler and spooled data.
+
+use std::path::PathBuf;
+
+use sqlx::sqlite::{SqliteConnectOptions, SqliteJournalMode, SqlitePoolOptions};
+
+use crate::service::create_runtime;
+use crate::services::spooler::{sql, BufferError};
+
+/// Truncates the spool file deleting all the persisted on-disk data.
+pub fn truncate(path: &PathBuf) -> Result<(), BufferError> {
+    let options = SqliteConnectOptions::new()
+        .filename(path)
+        .journal_mode(SqliteJournalMode::Wal);
+
+    let rt = create_runtime("truncator", 1);
```

Comment:
nit: adding the `spooler` prefix may help identify where it's coming from if we need to debug/profile (?)
```suggestion
    let rt = create_runtime("spooler-truncator", 1);
```

Reply:
This shouldn't run together with anything else, but just execute one command. 
I can rename it later if it's really will be necessary